scripts/dex-router/README.md

```diff
@@ -0,0 +1,35 @@
+## Overview
+
+This script bids to fulfill Limo orders via routing through on-chain liquidity.
+
+## Installation
+
+### npm
+
+```
+$ npm install
+```
+
+## Quickstart
+
+### Funding
+
+This script routes opportunities via on-chain liquidity, so your wallet will not need to hold any SPL tokens. However, it will need to hold some SOL to pay the gas cost.
+
+### Create Lookup Table
+
+Often, the on-chain liquidity routing instructions invoke a large number of accounts. As a result, the resulting set of instructions may be too large to fit into a single transaction. To minimize the size of the transaction, you can create a lookup table to store accounts commonly invoked in instructions like the Express Relay `SubmitBid` instruction using the [`solana address-lookup-table` CLI](https://docs.solanalabs.com/cli/usage#solana-address-lookup-table).
+
```

Comment:
Does the script work without setting these? If not, we may need more instructions here on what addresses we should put in the lookup tables.

Reply:
works without setting these